src/components/calls-page/calls-page.tsx

```diff
@@ -112,16 +112,16 @@ export const CallsPage = () => {
           setShouldReduceVolume(true);
         }, 1000);
       }
-    } else {
+    } else if (shouldReduceVolume) {
+      setShouldReduceVolume(false);
+    }
+
+    return () => {
       if (startTimeoutRef.current) {
```

Comment:
you probably wnat to check if `startTimeoutRef.current !== null`, in case the timeout id is 0.